pkg/minikube/bootstrapper/certs.go

```diff
@@ -49,17 +49,20 @@ func SetupCerts(cmd CommandRunner, k8s KubernetesConfig) error {
 	ip := net.ParseIP(k8s.NodeIP)
 	caCert := filepath.Join(localPath, "ca.crt")
 	caKey := filepath.Join(localPath, "ca.key")
-	publicPath := filepath.Join(localPath, "apiserver.crt")
-	privatePath := filepath.Join(localPath, "apiserver.key")
+	apiServerPublicPath := filepath.Join(localPath, "apiserver.crt")
+	apiServerPrivatePath := filepath.Join(localPath, "apiserver.key")
+	clientPublicPath := filepath.Join(localPath, "client.crt")
+	clientPrivatePath := filepath.Join(localPath, "client.key")
 
 	proxyClientCACert := filepath.Join(localPath, "proxy-client-ca.crt")
 	proxyClientCAKey := filepath.Join(localPath, "proxy-client-ca.key")
 	proxyClientPublicPath := filepath.Join(localPath, "proxy-client.crt")
 	proxyClientPrivatePath := filepath.Join(localPath, "proxy-client.key")
 
 	if err := generateCerts(
```

Comment:
Do you mind making this into a struct instead?  The arguments here are getting a little unwieldy.

Reply:
Yep... agree.

Reply:
@r2d4, I refactored, but chose a different approach. I found that it wasn't really necessary to pass all those paths into `generateCerts(...)`. It is just as easy to generate the paths _within_ that function and then the nasty function signature goes away.